source/templates/handlebars-basics.md

```diff
@@ -1,64 +1,72 @@
 Ember.js uses the [Handlebars templating library](http://www.handlebarsjs.com)
-to power your app's user interface. Handlebars templates are just like
-regular HTML, but also give you the ability to embed expressions that
-change what is displayed.
+to power your app's user interface. Handlebars templates contain static HTML and
+dynamic content inside what we call mustaches: `{{}}`.
 
-Ember takes Handlebars and extends it with many powerful features. It may
-help to think of your Handlebars templates as an HTML-like DSL for
-describing the user interface of your app. And, once you've told
-Ember.js to render a given template on the screen, you don't need to
-write any additional code to make sure it keeps up-to-date.
+Dynamic content inside a mustache is rendered with data-binding. This means if
+you update a property, your usage of that property in a template will be
+automatically updated to the latest value.
 
 ### Defining Templates
 
-The first thing you should change is your [application template](../the-application-template) that is created
-automatically for you and is displayed when your app loads.
-
-Next, you can define templates in the `app/templates` folder. By default,
-a route will render a template with the same name as the route.
+Templates in an Ember CLI app are usually stored in the `app/templates` folder.
+By default, a route will render a template with the same name as the route. For
+example this template would render for the `kittens` route:
 
 ```app/templates/kittens.hbs
 <h1>Kittens</h1>
 <p>Kittens are the cutest!</p>
 ```
 
-If you would like to create a template that is shared across many areas of your site, you should investigate [components](../../components/defining-a-component/).
+The `app/templates/application.hbs` file is the main template for your
+application. If you have a new Ember app, you can change the contents of that
+file and that content should appear in the browser. As a new user, you
+may want to experiment with content in the application template before diving
+in to routing.
+
+If you want to learn more about how routes and templates work together in
+Ember, jump to
+[The Application Route](../../routing/defining-your-routes/#toc_the-application-route)
+section of the guides.
```

Comment:
I agree with the sentiment. With more time a linear flow might be perfected, but this seems reasonable.

We've just spent a paragraph talking about the application template, and before that introduced a `kitty.hbs` template they basically have no idea how to use. And we about to suggest they try using the `if` helper without knowing how to provide properties to their templates.

IMO this is an excellent point to bridge them, and to hint at where they can find answers if the are confused by how to provide a `firstName` in the following sections.